Public/Src/Tools/ContentPlacementAnalysisTools/ContentPlacementAnalysisTools.Core/ContentPlacementAnalysisTools.Core.dsc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Tools.ContentPlacement.Core {

```

Comment:
>Tools [](start = 10, length = 5)

Do you mind removing "Tools" from the namespace since you are already in the module BuildXL.Tools it is a bit double for consumers...
i.e.

`importFrom("BuildXL.Tools").Tools.ContentPlacement...`